### DISCUSSION TITLE
Drastically decrease wait time between Arenas

### DISCUSSION THREAD

**Discussion by TotallyNotMyAccount** [chromiecraft/chromiecraft#6708] [Suggestions]
Currently there's a 5 minute enforced wait time between matches when teams have played each other previously and/or the rating difference between them is too large. This might be more reasonable if the server had a big enough population to support something like that, but currently it's just a flat 5 minute wait time after almost every match. What sounds harmless enough at first glance means in practice that if you only want to do the 10 games required to actually get arena points, you are looking at 50 minutes of pure wait time. Add in another minute of prep time before the actual match and then off we go to an arena match that’s often shorter than 2 minutes. Not only is that silly, it also destroys arena participation. 
Thus my suggestion: Decrease the wait time to 20-40 seconds. Keeping it around in a much reduced fashion ensures you are still preferably paired with teams you haven’t played before and those closer in rating to you. Now, I am aware that the other function of this lockout is to make wintrading harder, so if this is (or becomes) a major concern, the solution could be to go back to the 5 minutes during off hours. This isn’t exactly a perfect fix, as it would impact people from non-majority time zones, but those times are dead for legitimate players anyway and over all this still would be a massive improvement for the server.

**Comment by tbcxd** [comment]
I have way too many hours of just waiting, only to face the same team over and over again.
Make win trading an Ip/Permanent ban offense.

**Comment by Hurgon** [comment]
At the very start of the season it took over 2 hours to cap 10 games, more then it takes to clear whole SSC now. It was changed to 5 minutes, which in itself was a welcome change, but i also share the sentiment with a lot of other pvpers around - its still too much. For example yesterday we faced team in 100 rating range from us and we all still had to to wait 5 minutes each time. Reducing wait time could lead to more arena activity due to less downtime spent on doing nothing. Also last few changes lead to slight increase in pvp activity, especially rated arena and skirmishes, lest keep up the good trend!

**Comment by Amrx2** [comment]
To be brought forward after the weekly restart.
